config-templates/README.md

```diff
@@ -1,6 +1,6 @@
-#### Physics and Computer Science
+#### Computer Science
 
-If you want to get both physics and computer science content working, you'll need two copies of the `segue-config.properties` file. For consistency, use `segue-config.phy.properties` and `segue-config.cs.properties`.
+If you want to get computer science content working, you'll need two copies of the `segue-config.properties` file. For consistency, use `segue-config.cs.properties`.
```

Comment:
Is this correct? Do you actually need two copies? I would have thought you only need one (`segue-config.cs.properties`).

Also, if we're getting rid of the Physics content, does that mean that this setup can be simplified?

Reply:
Yes, you are correct. Will change this accordingly